src/edgepi/adc/adc_multiplexers.py

```diff
@@ -85,8 +42,9 @@ def validate_channels_allowed(channels: list, rtd_enabled: bool):
         if rtd_enabled
         else AllowedChannels.RTD_OFF.value
     )
-    for ch in channels:
-        if ch not in allowed_channels:
+    for chan in channels:
+        if chan not in allowed_channels:
             raise ChannelNotAvailableError(
-                f"Channel 'AIN{ch.value}' is currently not available. Disable RTD in order to use."
+                f"Channel 'AIN{chan.value}' is currently not available. "
+                "Disable RTD in order to use."
             )
```

Comment:
i would try to avoid nesting for a cleaner look:

```python
if any(chan not in allowed_channels for chan in channels):
    raise ChannelNotAvailableError(
        f"Channel 'AIN{chan.value}' is currently not available. "
        "Disable RTD in order to use."
    )
```

Reply:
Good idea!

https://github.com/EdgePi-Cloud/edgepi-python-sdk/pull/424/commits/accfb67f4fc0d8661d16970313515c90df6203d6